air-script/tests/aux_trace/aux_trace.masm

```diff
@@ -68,6 +95,16 @@ proc.compute_evaluate_integrity_constraints
     padw mem_loadw.4294900202 movdn.3 movdn.3 drop drop ext2mul
 end # END PROC compute_evaluate_integrity_constraints
 
+# Procedure to evaluate each integrity constraints.
+#
+# All the 2 main and 4 auxiliary constraints are computed.
+# The result constriants are evaluated in natural order, and their values are pushed to the stack.
+# The natural order is defined by `(trace, row, column)`, and the final evaluation is in stack-order.
+#
+# Input: [...]
+# Output: [(r_1, r_0)*, ...]
+# where: (r_1, r_0) is the quadratic extension resulting from the boundary constraint evaluation.
```

Comment:
nit: should probably say "quadratic extension element" rather than just "quadratic extension".

Reply:
chaged